packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -130,7 +125,7 @@ abstract final class FlutterOptions {
 }
 
 /// flutter command categories for usage.
-class FlutterCommandCategory {
+abstract final class FlutterCommandCategory {
```

Comment:
This one also smells like a new enum with a cool `.toString()` implementation :/